modules/ipc/methods/eth_sendTransaction.js

```diff
@@ -36,12 +36,13 @@ module.exports = class extends BaseProcessor {
             try {
                 _.each(payload.params[0], (val, key) => {
                     // if doesn't have hex then leave
-                    if (_.isString(val)) {
-
+                    if (!_.isString(val)) {
+                        throw this.ERRORS.INVALID_PAYLOAD;
+                    } else {
                         // make sure all data is lowercase and has 0x
-                        if (val) val = `0x${val.toLowerCase().replace('0x', '')}`;
+                        if (val) val = `0x${val.toLowerCase().replace(/^0x/igm, '')}`;
```

Comment:
in this context, the `g` and `m` flags seems to be redundant. As the regexp starts with `^`.